src/View/View.ts

```diff
@@ -28,6 +30,10 @@ export class View {
             this.cursorImage.data[i+3] = 0xFF;
         }
     }
+
```

Comment:
Maybe add a DrawableObject class to the View, that contains a point and an image. Then the controller can just pass drawableObjects for each button, instead of passing the whole toolbar object.

Reply:
I like this idea, but would like to revisit it later when other drawableObjects will be added instead of just the toolbar buttons.